scripts/geth-dev/start.sh

```diff
@@ -0,0 +1,3 @@
+geth --datadir /geth/.lpTest init /geth/genesis.json
```

Comment:
It wouldn't be crazy to move to a convention of `.livepeer/` as the default datadir and `.livepeer/testnet` as the default testnet data dir. The `.lpTest` feels arbitrary and like at some point we should move to something more professional.